config/spring19_seminar/cola_modified.conf

```diff
@@ -0,0 +1,3 @@
+include "../defaults.conf"
+
+cola_rel_path = CoLA2
```

Comment:
Add an example of this to defaults.conf, and add a comment there explaining what this does and why/how you should be careful with it.